### DISCUSSION TITLE
I'm not sure I fully understand the licensing. Can I use the community edition for SMB without restrictions and later upgrade to business as needed?

### DISCUSSION THREAD

**Discussion by tracure1337** [gravitl/netmaker#1784] [General]
I'm not sure I fully understand the licensing. Can I use the community edition for SMB without restrictions and later upgrade to business as needed?

Thanks for helping me out ;-)

**Comment by cameronts** [comment]
Absolutely!  You can also get a Free Tier of the Enterprise Edition if you'd like and that would allow for many SMBs to run the full business version while the business scales up.
